.github/workflows/ci.yml

```diff
@@ -138,6 +138,16 @@ jobs:
           BUNDLEMON_PROJECT_APIKEY: ${{ secrets.BUNDLEMON_PROJECT_APIKEY }}
         run: yarn bundlemon
 
+      - name: Deploy staging app to Firebase
+        if: github.ref == 'refs/heads/master'
```

Comment:
Is this condition already checked on line 90? Also, why is the condition different than for the heroku deploy, which uses `github.event_name == 'push'` instead of `'pull_request'`?

Reply:
"> On line 90 we are checking a more general form of this condition

Oops. I missed the `||` 😞

> I'm not sure, but semantically the difference is that `github.event_name == 'push'` will run for _any_ push to a branch, whether it's `master` or something else, and whether or not there is a PR open for that branch.

:D So, since we would `never` push directly to `master`, there is no difference. 🙈

Reply:
> So, since we would `never` push directly to `master`, there is no difference. 🙈

Merging a PR to `master` counts as push event I believe.